### PR TITLE
ISPN-9654 Upgrade to protostream-4.2.2.Final

### PR DESCRIPTION
**Pull request by anistor**
https://issues.jboss.org/browse/ISPN-9654

### CODE REVIEW

File: build-configuration/pom.xml [hunk 1]
```diff
@@ -180,7 +180,7 @@
       <version.org.wildfly.core>5.0.0.Final</version.org.wildfly.core>
       <version.osgi>4.3.1</version.osgi>
       <version.pax.exam>4.11.0</version.pax.exam>
-      <version.protostream>4.2.1.Final</version.protostream>
+      <version.protostream>4.2.2-SNAPSHOT</version.protostream>
```
Comment (tristantarrant) [inline build-configuration/pom.xml]: Final, not Snapshot please

**Comment by gustavonalle** [comment]
LGTM once the SNAPSHOT is removed (and all tests pass)

**Comment by anistor** [comment]
Updated. Let's see CI

File: build-configuration/pom.xml (hunk 1 above)
Comment (anistor) [reply build-configuration/pom.xml]: Done. but CI does not love it yet. Need to re-run it.

**Comment by tristantarrant** [comment]
Conflicts

**Comment by anistor** [comment]
CI shows 2 failures in jcache tck. Those are present on master also.
And there are several other failures in partition handling tests that do not seem related. Let's see the next build results.

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**